docs/release_notes/v1.2.0.md

```diff
@@ -0,0 +1,302 @@
+# Spice v1.2.0 (Apr 28, 2025)
+
+Spice v1.2.0 is a major release which includes upgrades to DataFusion v45, Arrow 54, and DuckDB v1.2, including improvements in query peformance, and introduces parameterized query support across SQL and HTTP APIs and accelerated views. Numerous bug fixes and dependency upgrades further improve stability and performance.
```

Comment:
I don't know if we want to avoid the specific term major here as to not conflict with the meaning from semantic versioning.